dosocs2/scannerbase.py

```diff
@@ -255,21 +256,23 @@ class FileLicenseScanner(Scanner):
 
     def store_results(self, processed_files):
         licenses_to_add = []
-        for (file, license_names) in processed_files.iteritems():
-            licenses = []
-            for license_name in license_names:
+        licenses = []
+        extracted_text = []
+        for (file, scan_result) in processed_files.iteritems():
+            for key in scan_result:
                 license_kwargs = {
                     'conn': self.conn,
-                    'short_name': license_name,
+                    'short_name': key,
                     'comment': 'found by ' + self.name
                     }
                 lic = scanresult.lookup_or_add_license(**license_kwargs)
                 licenses.append(lic)
-            for license in licenses:
+                extracted_text.append(scan_result[key][3])
```

Comment:
What is ```scan_result[key][3]``` ? Please unpack the tuple into a named variable rather than using a numbered index to select the 3rd item